### DISCUSSION TITLE
「贡献指南过于冗长」问题的一种解决思路

### DISCUSSION THREAD

**Discussion by IcyDesert** [HITSZ-OpenAuto/hoa-v2#185] [Ideas]
我们也许可以将贡献分成两类，对每一类分别撰写一份最简单的指南。

- 只修改文字
  例如修改 README.md，可以直接在 GitHub 界面修改

- 涉及到文件上传
  既可以 clone 到本地，传文件以后再修改，也可以使用 GitHub 的 codespace 在线修改

另外对于大物实验仓库，贡献者可能因图片太多而不想 clone，有无必要单独给仓库写一份贡献指南？这种方法可否可以推广到一些其他的特殊仓库（虽然我也不知道有哪些）？

**Comment by longlin10086** [comment]
支持分类，可以像 [AUTO野生技术指南](https://hoa.moe/blog/auto-survival-guide/) 这样把拆解后的文档归在一起。
关于大物实验单独贡献指南我不太清楚里面大概有哪些内容，需要更详细的说明。

**Comment by IcyDesert** [comment]
从贡献者角度看，大物实验仓库包含的图片等大文档较多，如果用本地仓库修改，`git clone` 和 `git push` 速度可能较慢，贡献者可能没有足够的时间等待操作完成，相当于失去潜在贡献者；直接在 GitHub 界面上传，又不能在一个 commit 里完成多文件上传；所以我想可以推荐 GitHub 自带的 codespace 进行多文件的编辑，或者像 [这样引流](https://hoa.moe/docs/fresh-spring/phys1002/#%e4%b8%aa%e4%ba%ba%e5%ae%9e%e9%aa%8c%e4%bb%93%e5%ba%93%e6%94%b6%e9%9b%86)

这只是我的一点看法，欢迎讨论

**Comment by longlin10086** [comment]
clone 和 push 速度慢感觉是国内网络问题而非我们仓库问题，如果是使用 codespace 或 自己仓库 是否也会遇到相同网络问题呢？在网络通畅的条件下我觉得三种贡献方法都是ok的

**Comment by longlin10086** [comment]
怎么转到 disscussion 了，这样做是否更难被看到
